scitbx/graph/tardy_tree.py

```diff
@@ -146,7 +145,9 @@ def cmp_clusters(a, b):
           if (fb): return 1
       if (len(a) > len(b)): return -1
       if (len(a) < len(b)): return 1
-      if (len(a) != 0): return cmp(a[0], b[0])
+      if len(a):
+        if a[0] > b[0]: return 1
+        if a[0] < b[0]: return -1
```

Comment:
Duplicate `cmp`
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.